### DISCUSSION TITLE
Snake neck customization

### DISCUSSION THREAD

**Discussion by mawilson** [BattlesnakeOfficial/feedback#220] [Feature Requests]
Would it be possible to add another dimension to snake customization, the neck? As in, the body part one immediately behind the head. This would be perfect for something like the trans rights scarf, & could even lean into the 'clothing' idea with different patterns & color schemes (also good for flags).

**Comment by andrewdmaclean** [comment]
Hey @mawilson 👋 

Sorry this response took so long 😓 

At the moment we have a new different ideas for additional customization options like you've presented, but none of them are in our pipeline for the near future.

Going to make this as feedback wanted for now!

Sorry again for the delay 🙏
